api/nativeMessaging/native_messaging_example_host_python.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+# https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Native_messaging
+# https://github.com/mdn/webextensions-examples/pull/157
+# Note that running python with the `-u` flag is required on Windows,
+# in order to ensure that stdin and stdout are opened in binary, rather
+# than text, mode.
+
+import sys
+import json
+import struct
+
+try:
+    # Python 3.x version
```

Comment:
Do you really want to support Python 2x at this point?

Reply:
It is possible that individuals in the wild are still running Python 2x.

Reply:
> While people may be using Python 2, to my knowledge Google no longer supports it.

I am not sure how Google is relevant to Python 2 run on the users' machine?

Reply:
MDN includes Python 2 code in their Native Messaging examples for a reason. I would ask why in the PR https://github.com/mdn/webextensions-examples/pull/157 or file an issue asking for clarity as to why, for consistency.